x-pack/plugin/sql/qa/src/main/resources/constant-keyword.csv-spec

```diff
@@ -0,0 +1,131 @@
+// To mute tests follow example in file: example.csv-spec
+
+//
+// Tests testing field alias (introduced in ES 6.4)
+//
+
+// filtering
+
+filterEquals
+SELECT extra_gender g FROM "test_emp_copy" WHERE g = 'M' LIMIT 5;
+
+    g     
+---------------
+M              
+M              
+M              
+M              
+M   
+
+;
+
+filterNotEquals
+SELECT COUNT(*) AS c FROM "test_emp_copy" WHERE extra_gender <> 'M';
+
+    c:l     
+---------------
+0
+;
+
+aggWithNullFilter
+SELECT COUNT(*) count FROM test_emp_copy WHERE extra_gender IS NOT NULL;
+
+     count:l     
+---------------
+100
+;
+
+functionOverAlias
+SELECT BIT_LENGTH(extra_gender) bit FROM test_emp_copy ORDER BY extra_gender LIMIT 1;
```

Comment:
If we have 2 values Male/Female that should be changed.

Reply:
A constant_keyword field has only one value.